### DISCUSSION TITLE
Error during compiling SU2:feature_TMZHB (had the same error on 2 different machines)

### DISCUSSION THREAD

**Discussion by koraysevinc** [su2code/SU2#1645] [Q&A]
**Describe the bug**
I have 3 different versions of SU2 on my machine and they all work (master branch, feature_turbomachinery_restruct_singlezone, feature_turbomachinery).
I wanted to try feature_TMZHB in order to study Harmonic Balance problems. However during make process I faced an error. I tried it on my laptop as well and faced exactly the same error. It occurred during compiling of geometry_structure.cpp.

I don't know how to solve this issue. Thanks for your help in advance.
Koray

**To Reproduce**
Here's my configure outputs:

Build Configuration Summary:

    Source code location: /home/koray/SU2_TMZHB/SU2
    Install location: 	  /usr/local
    Version:              6.0.1
    C++ Compiler:         g++
    C Compiler:           gcc
    Preprocessor flags:   
    Compiler flags:       -g -O2
    Linker flags:         
    MPI support:          no
    Metis support:        no
    Parmetis support:     no
    TecIO support:        yes
    CGNS support:         yes
    HDF5 support:         no
    SZIP support:         no
    ZLIB support:         no
    Mutation++ support:   no
    Jsoncpp support: 	  no
    LAPACK support:       no
    Datatype support:
        double              yes
        complex             no
        codi_reverse        no
        codi_forward        no

    External includes:    -DHAVE_CGNS -I$(top_srcdir)/externals/cgns -I$(top_srcdir)/externals/cgns/adf -DHAVE_TECIO -I$(top_srcdir)/externals/tecio/include 
    External libs:        $(top_builddir)/externals/cgns/libcgns.a $(top_builddir)/externals/tecio/libtecio.a 

    Build SU2_CFD:        yes
    Build SU2_DOT:        yes
    Build SU2_MSH:        yes
    Build SU2_DEF:        yes
    Build SU2_SOL:        yes
    Build SU2_GEO:        yes
    Build Py Wrapper:     no
################

Here are the error lines during make process:

  CXX      ../src/libSU2_a-dual_grid_structure.o
  CXX      ../src/libSU2_a-geometry_structure.o
../src/geometry_structure.cpp: In member function ‘virtual void CPhysicalGeometry::UpdateTurboVertex(CConfig*, short unsigned int, short unsigned int)’:
../src/geometry_structure.cpp:14365:20: error: ‘HUGE’ was not declared in this scope
14365 |   su2double max = -HUGE;
      |                    ^~~~
make[1]: *** [Makefile:857: ../src/libSU2_a-geometry_structure.o] Error 1
make[1]: Leaving directory '/home/koray/SU2_TMZHB/SU2/Common/lib'
make: *** [Makefile:391: all-recursive] Error 1
###

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04.2 LTS
 - C++ compiler and version:  gcc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0
 - MPI implementation and version: mpiexec (OpenRTE) 4.0.3
 - SU2 Version: SU2 (ver. 6.0.1 "Falcon")

**Comment by koraysevinc** [comment]
Hi, I am still facing this issue. Does anyone have an idea about that? Thanks.

../src/geometry_structure.cpp:14365:20: error: ‘HUGE’ was not declared in this scope
14365 | su2double max = -HUGE;

**Comment by pcarruscag** [comment]
That branch is very old. The person that created it is probably not working on su2 anymore.
If you just want to make it compile somehow, I guess 1e30 is a suitable replacement for HUGE.
